thrift.js

```diff
@@ -204,13 +199,13 @@ Thrift.prototype._asyncParseIncludedModules = function _asyncParseIncludedModule
             return handleCb(undefined);
         }
         var includeFilename = path.join(dirname, def.id);
-        model._asyncParse(includeFilename, true, function (err) {
```

Comment:
Passing `allowIncludeAlias` around as argument is used because it was always `true` except for the parent source before the browser support changes. These changes would use `this.allowIncludeAlias` instead, isn't it a breaking change?

Reply:
This is a bug fix. Thank you!

Reply:
Ah ok nice